src/java/test/org/apache/zookeeper/test/LoadFromLogTest.java

```diff
@@ -449,50 +267,25 @@ public void testRestoreWithTransactionErrors() throws Exception {
         // Force snapshot and restore
         zks.takeSnapshot();
         zks.shutdown();
-        f.shutdown();
+        stopServer();
 
         zks = new ZooKeeperServer(tmpDir, tmpDir, 3000);
         SyncRequestProcessor.setSnapCount(10000);
-        f = ServerCnxnFactory.createFactory(PORT, -1);
-        f.startup(zks);
-        Assert.assertTrue("waiting for server being up ", ClientBase
-                .waitForServerUp(hostPort, CONNECTION_TIMEOUT));
-
-        f.shutdown();
-        zks.shutdown();
+        startServer();
     }
 
     /**
      * Verify snap/log dir create with/without autocreate enabled.
      */
     @Test
     public void testDatadirAutocreate() throws Exception {
-        ClientBase.setupTestEnv();
```

Comment:
not sure if it matters here and in the following test method but the snapcount is different after the current patch.